11/client/src/javascripts/app.js

```diff
@@ -0,0 +1,52 @@
+let wd = require('./wd.js');
+
+window.onload = function () {
+    wd.createRoot('app')
+        .routes([
+            { url: '/', templateUrl: 'routes/home.html' },
+            { url: '/sign-in', templateUrl: 'routes/sign-in.html' }
+        ])
+        .component("RouterLink", {
+            template: "<a href=\"\"><\/a>",
+            beforeMount: function (app, element, componentData) {
+                let replacement = document.createElement("a");
+                replacement.href = element.getAttribute("href");
```

Comment:
Please, see my comments for task 10 pull request. The same can be improved here.